codeland.html

```diff
@@ -767,3 +843,35 @@ <h2 class="codeland-section__title align-center">
     </div>
   </section>
 </div>
+
+<script>
+  const articleButt = document.querySelector('.codeland-livestream__speaker-info div button');
+  const closeButt = document.querySelector('.codeland-article-sidecar header button');
+  const sidecarDiv = document.querySelector('.codeland-article-sidecar');
+
+  function removeSidecarIframe() {
+    const sidecarEmbed = sidecarDiv.querySelector('iframe');
+    if (sidecarEmbed !== null) {
+      sidecarDiv.removeChild(sidecarEmbed);
```

Comment:
Just spitballing here: but what's the performance implication of adding and removing the entire div versus potentially just rewriting the URL/location? Would that refresh it?

Reply:
The div always stays in place and it's only the iframe that's being replaced, but I'm still not sure of the performance implications TBH. It's a good point to keep in mind!

I think this has to do with the browser implementation of iframes. Since it's all within the same browser tab I'd guess the browser is smart enough to use the locally cached assets (images, CSS, JS, etc) regardless if the request is performed within an iframe, but it's all speculation at this point.

We can reuse the same panel for the Connect panels that pull out from the side, so this is something I'll definitely look into!